components/icm42670/.eil.yml

```diff
@@ -0,0 +1,23 @@
+---
+components:
+  - name: icm42670
+    description: Driver for TDK ICM-42670-P IMU (found on ESP-RS board)
+    group: imu
```

Comment:
You need to describe this group by addinig it to `devtools/groups.yml`.
Also describe yourself in `devtools/persons.yml`.